server/routes.js

```diff
@@ -0,0 +1,254 @@
+var _fs             = require("fs")
+let denodeify       = require("denodeify")
+let config          = require("../config")
+let users           = require("../users")
+
+const fs = {
+	readFile: denodeify(_fs.readFile)
+}
+
+module.exports = [
+	{
+		path:/^\/$/,
+		page: "page.hbs",
+		cache: true,
+		prerender: ["/"],
+		context: (_, db) =>
+			aggregatePosts(db, { $sort: { timestamp: -1 } }, { $limit: 5 })
+				.then(fillAuthorInfo(db))
+				.then((posts) => {
+					posts.forEach((post) => post.short = true);
+					return db.posts.count()
+						.then((count) => ({
+							title: "",
+							posts,
+							pagination: {
+								right: count > 5 ? "/archive/2" : undefined
+							}
+						}))
+				})
+				.then(fillDefaultSidebar(db))
+	},
+	{
```

Comment:
No.  Different contexts entirely, IMO.

`() => {` comes from the fact that I write `function() {` on one line.

Separating objects in JSON is because I (in most cases) prefer a newline after each element in an array if it's filled with non-primitives.